systems/plants/RigidBodyManipulator.cpp

```diff
@@ -676,6 +676,45 @@ bool RigidBodyManipulator::collisionDetect( VectorXd& phi,
   return collisionDetect(phi, normal, xA, xB, bodyA_idx, bodyB_idx, ids_to_check, use_margins);
 }
 
+bool RigidBodyManipulator::potentialCollisions(VectorXd& phi,
+                                            MatrixXd& normal,
+                                            MatrixXd& xA,
+                                            MatrixXd& xB,
+                                            vector<int>& bodyA_idx,
+                                            vector<int>& bodyB_idx,
+                                            bool use_margins)
+{
```

Comment:
I would prefer calling potentialCollisionPoints outside potentialCollisions, passing in potential_collisions as an argument to the method and having potentialCollisions not return anything.